### DISCUSSION TITLE
Requests password for unlock automatically

### DISCUSSION THREAD

**Discussion by kaspersoerensen** [blacs30/bitwarden-alfred-workflow#111] [Q&A] [answered]
I am using the auto lock function, set for 120 mins. When the auto-lock kicks in, Alfreds pops up, promoting me to unlock the vault with my password. I'm not sure whether this is a bug or a feature, but it is rather annoying. I would think, it should just wait to request an unlock, till I am trying to access the items in the vault again?  

Have someone else stumbled across this issue?

**Comment by blacs30** [comment]
Thanks for reporting it. I have seen it myself but didn't have time to look into it yet.

**Comment by kaspersoerensen** [comment]
Short update: It seems to only happen on my mac running Mojave (10.14.6). I have another machine, running Catalina, and I have not yet observed the problem there.

**Comment by blacs30** [comment]
After reinstalling the auto lock the pop up is gone. I've set the timeout to 1 minute though for testing. Will set it back to a normal 60 minutes and see how it behaves then.

**Comment by blacs30** [comment, marked as answer]
I found the reason for this behaviour:
If Bitwarden is locked but the auto sync daemon is enabled and starts to run then it brings up the Bitwarden unlock message.
It's basically by design because the background sync daemon uses the same function as if the sync would be triggered via workflow - it checks for the unlock status and if it's locked it asks to unlock.
